packages/sentinel/src/api/index.test.ts

```diff
@@ -15,6 +15,9 @@ jest.mock('defender-base-client');
 jest.mock('aws-sdk');
 jest.mock('axios');
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { createAuthenticatedApi } = require('defender-base-client');
```

Comment:
Why is this not `import`ed? If it has to do with the order of `mock -> import`, I'm 90% sure it still mocks it

Reply:
It does, but typescript complained about the types, hence the require.